src/styles/index.scss

```diff
@@ -1,3 +1,147 @@
 body {
-  margin: 0;
+  $card-border: #f3f3f3;
+  $color-black: #060b35;
+  $color-gray: #616070;
+  $color-blue: #00acdc;
+  $color-white: #fff;
+  $gap: 4px;
+  
+  body,
+  p,
+  a,
+  img,
+  div {
+    margin: 0;
+    box-sizing: border-box;
+    font-family: Roboto, Helvetica, sans-serif;
+  }
+  
+  .card {
+    width: 200px;
+    height: 406px;
+    padding: 32px 16px 16px;
+    border-radius: 5px;
+    border: 1px solid $card-border;
+  
+    &__image {
+      display: block;
+      width: 160px;
+      height: 134px;
+      margin: 0 auto;
+    }
+  
+    &__title {
+      margin-top: 40px;
+      margin-bottom: 0;
+      font-style: normal;
+      font-size: 12px;
+      line-height: 18px;
+      font-weight: 500;
+      color: $color-black;
+      overflow: hidden;
+    }
+  
+    &__code {
+      margin-top: 4px;
+      font-size: 10px;
+      font-style: normal;
+      font-weight: 400;
```

Comment:
Not fixed 
![2023-10-03_13h32_06](https://github.com/mate-academy/layout_product-cards/assets/61904995/abd39c0d-03d5-4e6e-ba64-7dec841a8815)